### PR TITLE
IDEMPIERE-4607 Cannot export windows via 2pack if any field has a Fie…

### PR DESCRIPTION
**Pull request by d-ruiz**
…ld Group

https://idempiere.atlassian.net/browse/IDEMPIERE-4607

### CODE REVIEW

**Review by CarlosRuiz-globalqss — CHANGES_REQUESTED** [review]
See comments in ticket

File: org.adempiere.base/src/org/compiere/util/DisplayType.java [hunk 1]
```diff
@@ -363,6 +363,35 @@ public static boolean isDate (int displayType)
...
+	{
+		if (DisplayType.List == displayType  || DisplayType.RadiogroupList == displayType
+				|| DisplayType.ChosenMultipleSelectionList == displayType)
+			return true;
+		
+		IServiceReferenceHolder<IDisplayTypeFactory> cache = s_displayTypeFactoryCache.get(displayType);
+		if (cache != null) {
+			IDisplayTypeFactory service = cache.getService();
+			if (service != null)
+				return service.isDate(displayType);
```
Comment (hengsin) [inline org.adempiere.base/src/org/compiere/util/DisplayType.java:383]: typo

Comment (d-ruiz) [reply org.adempiere.base/src/org/compiere/util/DisplayType.java]: Fixed

File: org.adempiere.base/src/org/adempiere/base/IDisplayTypeFactory.java [hunk 2]
```diff
@@ -29,6 +29,7 @@
 	public Integer getDefaultPrecision(int displayType);
 	public boolean isText(int displayType);
 	public boolean isDate (int displayType);
+	public boolean isList (int displayType);
```
Comment (CarlosRuiz-globalqss) [inline org.adempiere.base/src/org/adempiere/base/IDisplayTypeFactory.java:32]: This change breaks the plugins that use the interface.

I suggest the following change for better backward compatibility:
[default_interface.txt](https://github.com/idempiere/idempiere/files/5743146/default_interface.txt)

**Review by CarlosRuiz-globalqss — APPROVED** [review]

**Pull request merged**